docs/source/models/regnet.rst

```diff
@@ -0,0 +1,37 @@
+RegNet
+======
+
+.. currentmodule:: torchvision.models
+
+The RegNet model is based on the `Designing Network Design Spaces
+<https://arxiv.org/pdf/2003.13678.pdf>`_ paper.
```

Comment:
Let's link to the archive page  instead of the `.pdf` file, because this can trigger undesired downloads.

Reply:
got it